src/components/DuffelAncillaries/seats/Legend.tsx

```diff
@@ -47,6 +47,12 @@ export const Legend: React.FC<LegendProps> = ({ symbols }) => (
       </span>
       Unavailable
     </span>
+    <span className="seat-map__legend-item">
+      <span className="seat-map__legend-seat" aria-label="Restricted seat">
+        <Icon name="close" size={14} />
+      </span>
+      Restricted
+    </span>
```

Comment:
Jw, if the visual is the same, why not do something like this:

```suggestion
      Unavailable/Restricted
    </span> 
```

?

Reply:
Good idea! I think it's ok to keep them the same for now, will see if anyone asks